### PR TITLE
Refactor RegistrationsController#new to redirect if current_user

### PR DESCRIPTION
**Pull request by smoil**
Redirect to after_login_path if a user is currently logged in.

I'm making a few assumptions about the desired workflow so please feel free to let me know if this is off-base or if you have any questions.

### CODE REVIEW

File: app/controllers/thincloud/authentication/registrations_controller.rb [hunk 1]
```diff
@@ -8,6 +8,10 @@ class RegistrationsController < ApplicationController
     helper "thincloud/authentication/registrations"
 
     def new
+      if current_user
+        redirect_to after_login_path, notice: "You are already registered."
```
Comment (bousquet) [inline app/controllers/thincloud/authentication/registrations_controller.rb:12]: Do you need main_app.after_login_path or are there wrappers local to the engine that wrap the configuration values?

Comment (smoil) [inline app/controllers/thincloud/authentication/registrations_controller.rb:12]: They should be local, check out #create https://github.com/newleaders/thincloud-authentication/blob/master/app/controllers/thincloud/authentication/registrations_controller.rb#L40

**Comment by bousquet** [comment]
Methinks this functionality warrants a test to prevent future regressions.

**Comment by smoil** [comment]
There is one right now.  What else would you like to see in it?

**Comment by bousquet** [comment]
Oh, my bad.  Didn't see it.

**Pull request merged**